packages/cli/src/util/projects/update-project.ts

```diff
@@ -1,26 +1,16 @@
 import Client from '../client';
-import type { JSONObject, ProjectSettings } from '@vercel-internals/types';
-
-interface ProjectSettingsResponse extends ProjectSettings {
-  id: string;
-  name: string;
-  updatedAt: number;
-  createdAt: number;
-}
```

Comment:
Seems like removing this will result in a return type of `any`.

Is that intentional?

Reply:
Not the case. Return type is `Promise<Project>`:

<img width="792" alt="Screenshot 2023-07-03 at 2 35 26 PM" src="https://github.com/vercel/vercel/assets/71256/fff3ca77-93d4-4c21-ad77-0685e79b611e">